components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.jsx

```diff
@@ -36,11 +37,21 @@ const EXCLUDED_PERMISSIONS = [
     Permissions.LIST_PRIVATE_TEAMS,
 ];
 
+const GUEST_INCLUDED_PERMISSIONS = [
+    Permissions.CREATE_PRIVATE_CHANNEL,
+    Permissions.EDIT_POST,
+    Permissions.DELETE_POST,
+    Permissions.ADD_REACTION,
+    Permissions.REMOVE_REACTION,
+];
```

Comment:
Adding a newly exposed guest permission to `GuestPermissionsTree` adds the permission to the UI for both of the`PermissionTeamSchemeSettings` and `PermissionSystemSchemeSettings` components, but both are *also* required to update their `GUEST_INCLUDED_PERMISSIONS` arrays otherwise the new checkbox does nothing.

I think both of the `Permission*SchemeSettings` components should import this array from `GuestPermissionsTree` and then `GuestPermissionsTree` component is the source of truth and their behaviour is more likely to be kept in lockstep.

Reply:
`EXCLUDED_PERMISSIONS` is also duplicated across two files `permission_system_scheme...` and `permission_team_scheme..` 

I think both `EXCLUDED_PERMISSIONS` and `GUEST_INCLUDED_PERMISSIONS` can be added to the `permissions_tree` file as exported constants

Reply:
Agreed